frontend/src/app/main/data/workspace.cljs

```diff
@@ -242,9 +251,10 @@
     (update [_ state]
       (let [page-id (or page-id (get-in state [:workspace-data :pages 0]))
             local   (-> (:workspace-local state)
-                        (dissoc :edition)
-                        (dissoc :edit-path)
-                        (dissoc :selected))]
+                        (dissoc
```

Comment:
Technically is better call dissoc multiple times than a single call with varargs.